patches/tModLoader/Terraria/ModLoader/ModLoader.cs

```diff
@@ -374,16 +374,11 @@ internal static void MigrateSettings()
 			if (LastLaunchedTModLoaderVersion < new Version(0, 11, 7, 5))
 				showMemoryEstimates = true;
 
-			if (BuildInfo.Purpose == BuildInfo.BuildPurpose.Alpha && LastLaunchedTModLoaderAlphaSha != BuildInfo.CommitSHA) {
+			if (BuildInfo.Purpose == BuildInfo.BuildPurpose.Dev && LastLaunchedTModLoaderAlphaSha != BuildInfo.CommitSHA || LastLaunchedTModLoaderVersion != BuildInfo.tMLVersion) {
```

Comment:
Should we change `LastLaunchedTModLoaderAlphaSha` to remove the `Alpha` and use it everywhere? The other places these are checked are a bit inconsistent with this one.

Reply:
I'll have to defer to Jopo on this one for eliminating the LastLaunchedAlpha fully - the setup for how the "Show What's New" prompt is quite integrated with it, so I'd rather not break that right now